contracts/interfaces/Address.sol

```diff
@@ -0,0 +1,252 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.6.2 <0.8.0;
```

Comment:
Feel free to add `contracts/interfaces/` to `.solhintignore` if we're only using that dir for copy-pasted scripts.